launcher/Version.cpp

```diff
@@ -123,8 +123,19 @@ QDebug operator<<(QDebug debug, const Version& v)
         first = false;
     }
 
-    debug.nospace() << " ]"
-                    << " }";
+    debug.nospace() << " ]" << " }";
 
     return debug;
 }
+
+bool checkMcVersions(std::list<Version> filter, QStringList value)
+{
+    bool valid = false;
+    for (auto mcVersion : filter) {
+        if (value.contains(mcVersion.toString())) {
+            valid = true;
+            break;
+        }
+    }
+    return filter.empty() || valid;
+}
```

Comment:
(Nitpick) This is an odd home and name for this function now it has been moved - it is not specific to mc versions and feels like a util function just inserted into the header file for no reason other than being version-related. I think it would make more sense as member function named Version::isAnyOf.

Also I can't help but notice the code does the same as
```c++
for (auto mcVersion : filter)
    if (value.contains(mcVersion.toString()))
        return true;

return filter.empty();
```
Or, perhaps easier to read:
```c++
if (filter.empty())
    return true;

for (auto mcVersion : filter)
    if (value.contains(mcVersion.toString()))
        return true;

return false;
```

Reply:
Moved on ModFilterWidget::Filter as it makes sense to be there